src/func.d

```diff
@@ -4486,7 +4486,7 @@ public:
         return this;
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
         // GCC requires the (char*) casts
         return (tok != TOKfunction) ? cast(char*)"delegate" : cast(char*)"function";
```

Comment:
one thing at a time, guys. I don't care to post 5000 line PRs, and nobody will review them.